src/NUnitTestAdapterTests/NavigationDataTests.cs

```diff
@@ -1,53 +1,32 @@
 ﻿using System.IO;
 using NUnit.Framework;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Logging;
 
 namespace NUnit.VisualStudio.TestAdapter.Tests
 {
     using Fakes;
 
-    [Category("Navigation")]
-    public class NavigationDataTests
+    class NoSourceMessageTrap : IMessageLogger
     {
-        TestConverter testConverter;
-
-        const string Prefix = "NUnit.VisualStudio.TestAdapter.Tests.NavigationTestData";
-
-        [SetUp]
-        public void SetUp()
+        public void SendMessage(TestMessageLevel testMessageLevel, string message)
         {
-            
-            testConverter = new TestConverter(
-                new TestLogger( new MessageLoggerStub(), 0), 
-                Path.Combine(TestContext.CurrentContext.TestDirectory, "NUnit.VisualStudio.TestAdapter.Tests.dll"));
+            Assert.That(message, Does.Not.Contain("No source data found for"));
```

Comment:
Since this is only a unit test, I would say that it's a sufficient test to verify that we get the correct navigation, I don't think we need to look at the messge at all, so we don't really need this extra logger.